sst/stacks/ToDontStack.ts

```diff
@@ -1,33 +1,31 @@
-import { 
+import {
   Api,
-  Function,
   StackContext,
-  Topic 
+  Table,
+  Topic
 } from "@serverless-stack/resources";
-import { 
-  AttributeType,
-  BillingMode,
-  StreamViewType,
-  Table 
-} from "aws-cdk-lib/aws-dynamodb";
-import { PolicyStatement } from "aws-cdk-lib/aws-iam";
-import { StartingPosition } from "aws-cdk-lib/aws-lambda";
-import { DynamoEventSource } from "aws-cdk-lib/aws-lambda-event-sources";
 
-export function ToDontStack({ stack }: StackContext) {  
+export function ToDontStack({ stack }: StackContext) {
+  // create our SNS topic
+  const topic = new Topic(stack, "topic");
+
   // create our DynamoDB table
-  const ddb = new Table(stack, "Table", {
-    billingMode: BillingMode.PAY_PER_REQUEST,
-    partitionKey: {
-      name: "pk",
-      type: AttributeType.STRING,
+  const ddb = new Table(stack, "table", {
+    fields: {
+      pk: "string",
     },
-    stream: StreamViewType.NEW_AND_OLD_IMAGES,
+    primaryIndex: { partitionKey: "pk" },
+    stream: "new_and_old_images",
+    consumers: {
+      fanout: {
+        function: {
+          handler: "functions/fanout.handler",
+          bind: [topic],
+        },
+      }
+    }
```

Comment:
Using SST's table construct. And we don't need to import from `aws-cdk-lib`.

^ SST constructs covers the common use-cases (ie. adding ddb stream consumers) so users don't need to import from `aws-cdk-lib`.